lib/src/handle_access_token_response.dart

```diff
@@ -17,6 +17,54 @@ import 'authorization_exception.dart';
 /// amount of time.
 const _expirationGrace = const Duration(seconds: 10);
 
+/// Returns a function that that parses parameters from a response with either a JSON or URL-encoded body.
```

Comment:
I still don't want to bake in support for GitHub's unspecified URL-encoded format. The core package should only support the actual spec.